day11/prj.py

```
@@ -0,0 +1,126 @@
+
+def continue_blackjack():
+    """Handles the computer's turn to draw cards until reaching a score of at least 17."""
+    continue_q = input("Do you want to play a game of Blackjack? Type 'y' or 'n': ")
+    while continue_q not in ['y', 'n']:
+        continue_q = input("Please type 'y' or 'n': ")
+    return continue_q
+
+def calculate_winner():
+    """Calculates and displays the winner of the game."""
+    final_info()
+    if u_score == c_score:
```

Ruff (0.12.2)

Undefined name `u_score`

(F821)